portality/util.py

```diff
@@ -187,6 +187,35 @@ def get_full_url_safe(endpoint):
         app.logger.warning(f'endpoint not found -- [{endpoint}]')
         return None
 
+
 def no_op(*args, **kwargs):
     """ noop (no operation) function """
-    pass
\ No newline at end of file
+    pass
+
+
+def custom_timed_rotating_logger(file_name):
+    """Custom Logger to log to specified file name"""
+    import os
+    import logging
+    from logging.handlers import TimedRotatingFileHandler
+    # Create a logger
+    logger = logging.getLogger(os.path.splitext(file_name)[0])
+    logger.setLevel(logging.DEBUG)  # Set the logging level
+
+    # Get the user's home directory
+    user_home = os.path.expanduser("~")
```

Comment:
this is probably best suited to the `appdata` folder - currently in `~/appdata/doaj/`

Reply:
updated log dir to `~/appdata/ doaj` at `log_dir = os.path.join(user_home, 'appdata', 'doaj')`